CondFormats/SiPixelTransient/src/SiPixelTemplate.cc

```diff
@@ -2271,7 +2271,7 @@ void SiPixelTemplate::ysigma2(float qpixel, int index, float& ysig2)
 //! \param xsum - (input) 11-element vector of pixel signals
 //! \param xsig2 - (output) 11-element vector of x errors (squared)
 // ************************************************************************************************************
-void SiPixelTemplate::xsigma2(int fxpix, int lxpix, float sxthr, float xsum[11], float xsig2[11])
+void SiPixelTemplate::xsigma2(int fxpix, int lxpix, float sxthr, float xsum[BXSIZE], float xsig2[BXSIZE])
```

Comment:
I might be wrong (not expert of this code) - but it seems to me that BXSIZE is not `11` but `16`, from:
https://github.com/cms-sw/cmssw/blob/21ff8f03ca9c662943fa2b49f719f2ca54039606/CondFormats/SiPixelTransient/interface/SiPixelTemplateDefs.h#L30-L33

Reply:
@francescobrivio , in the header file it `BXSIZE` is used https://github.com/cms-sw/cmssw/blob/master/CondFormats/SiPixelTransient/interface/SiPixelTemplate.h#L321-L323 that is why I changed it to be BXSIZE. We should be consistent, if `11` is the right value then we should update the header file.

Reply:
I guess we need help from @cms-sw/trk-dpg-l2 to know which is the correct value? Also, maybe @mmusich can help here?

Reply:
@tvami @pmaksim1 are the experts of this code.... having said that, I tend to believe more to the header argument list.

Reply:
Yeah, I think it is ok to just go with`BXSIZE`